specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -220,7 +246,7 @@ Given these elements, the number of test cases for each scenario can be calculat
 
 **Alternate flows**
 
-- TODO
+- Calling a Team-scoped skill from a 1-to-1 or group chat conversation
```

Comment:
Not sure I get this flow, what happens when a user calls a Team-scoped skill from a 1:1 conversatio?

Reply:
resolved